pages/rotas.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <div v-if="cidsRoutes" class="routes-page__container">
+    <div class="routes-page__chips-container">
+      <v-chip
+        v-for="(item, index) in cidsRoutes"
+        :key="index"
+        class="routes-page__chip"
+        close
+        close-icon="mdi-close"
+        :color="cidsRoutes[index].color"
+        text-color="white"
+        @click="()=>updateSelectedCidRoute(index)"
+      >
+        {{index}}
+      </v-chip>
+    </div>
+    <div class="routes-page__steps">
+      <v-carousel v-model="model">
+        <v-carousel-item
+          v-for="(route, index) in cidsRoutes[selectedCidRoute].routes"
+          :key="index"
+        >
+          <v-card class="routes-page__card" height="90%">
+            <v-card-text>
+              <div class="font-weight-bold ml-8 mb-2">Itinerário {{ index + 1 }}</div>
+
+              <v-timeline align-top dense>
+                <v-timeline-item
+                  v-for="(step, indexStep) in route"
+                  :key="indexStep"
+                  :color="cidsRoutes[selectedCidRoute].color"
+                  small
+                >
+                  <div>
+                    <div class="font-weight-normal">
+                      <strong>{{ step.name }}</strong> @{{ step.time }}
+                    </div>
+                    <div>{{ step.address }}</div>
+                  </div>
+                </v-timeline-item>
+              </v-timeline>
+            </v-card-text>
+          </v-card>
+        </v-carousel-item>
+      </v-carousel>
+    </div>
+  </div>
+</template>
+
+<script>
+import api from './mockApi.json'
+
+export default {
+  name: 'RoutesPage',
+  data() {
+    return {
+      cidsRoutes: null,
+      model: 0,
+    }
+  },
+
+  async fetch() {
+    this.cidsRoutes = await api
+    this.selectedCidRoute = Object.keys(this.cidsRoutes)[0]
+  },
+
+  methods: {
+    updateSelectedCidRoute(index) {
+      this.selectedCidRoute = index
+      this.model = 0
+      this.model = 1
+      this.model = 0
+      // Até o momento o único jeito que encontrei de fazer o componente atualuzar para o slide certo
```

Comment:
🤔

Reply:
Quando eu mando um this.mode = alguma coisa ele emite um evento, que eu não sei qual é, pra renderizar de novo. Se eu não fizer isso, ele só atualiza a rota quando eu clico em uma das setinhas do lado

Reply:
depois a gente cria uma task pra investigar melhor isso então